doc/user_guide/eels.rst

```diff
@@ -149,14 +149,18 @@ Define the chemical composition of the sample
     >>> s.add_elements(('B', 'N'))
 
 
-In order to include the effect of plural scattering we provide the low-loss
-spectrum to :py:meth:`~._signals.eels.EELSSpectrum_mixin.create_model`:
+In order to include the effect of plural scattering, the component of the model is convolved with the loss loss spectrum in which case the low loss spectrum needs to be provided to :py:meth:`~._signals.eels.EELSSpectrum_mixin.create_model`:
 
 .. code-block:: python
 
     >>> m = s.create_model(ll=ll)
 
 
+.. NOTE::
+
+    In case of the convolution with the low loss spectrum, the fitted value of some of the parameter may seem to be unexpectedly small which can be misleading. Those paramater values are actually correct since the values of the low loss spectrum are usually much larger than 1. Indeed, the `convolution <https://en.wikipedia.org/wiki/Convolution>`_ gives the integral of the pointwise multiplication of the component function with the low-loss spectrum shifted over the component and it results that the function values of the convolved component are much larger than in the non-convolved case, which means that the fitted parameter values need to be very small to fit the data.
```

Comment:
I would remove this note because the reason why the numbers are small is that hyperspy assumes the dwell time of the ll and cl to be the same. We should rather fix that...

Reply:
Removed it. Possibly better to leave this for another PR. Now, we should be parsing the time correctly from DM files, so it should be fine to fix it indeed.